DQM/BeamMonitor/python/AlcaBeamMonitorHeavyIons_cff.py

```diff
@@ -7,7 +7,10 @@
 AlcaBeamMonitor.BeamFitter.TrackCollection = 'hiGeneralTracks'
 AlcaBeamMonitor.BeamFitter.TrackQuality    = ['highPurity']
 AlcaBeamMonitor.PVFitter.VertexCollection  = 'hiSelectedVertex'
-
+#Check if perLSsaving is enabled to mask MEs vs LS
+from DQMServices.Core.DQMStore_cfi import DQMStore
+if(DQMStore.saveByLumi):
+      AlcaBeamMonitor.perLSsaving=True
```

Comment:
Doing
```py
process.load("DQM.BeamMonitor.AlcaBeamMonitorHeavyIons_cff")
process.DQMStore.saveByLumi = True
```
leads to `AlcaBeamMonitor.perLSsaving` being `False`. ProcessModifiers is the best tool for customizing configuration parameters coherently across cfi/cff files.

Reply:
Thanks @makortel  but I don't understand the suggestion since this is not an Era modifier, but a configurtaion parameter that would be changed as  cmsDriver parameter
--customise_commands=process.DQMStore.saveByLumi = cms.untracked.bool(True) 
So, this variable is predefined a priori

Reply:
> a configuration parameter that would be changed as cmsDriver option in customise:
> --customise_commands=process.DQMStore.saveByLumi = cms.untracked.bool(True)
> So, this variable is predefined a priori

But it's not, as far as the execution of the python code in the generated configuration file is concerned. `cmsDriver` puts the `--customise_commands` at the end of the `cfg` file, which leads to the situation I (rather tersely) described: the `cff` file is processed first with the default value `DQMStore.saveByLumi`, and only after then `DQMStore.saveByLumi` is set to `True`, but that has no effect on the `AlcaBeamMonitor`.

You can try yourself with e.g.
```
# step2 of 140.56
cmsDriver.py step2_test --conditions auto:run2_data_promptlike_hi -s RAW2DIGI,L1Reco,RECO,ALCA:SiStripCalZeroBias+SiPixelCalZeroBias,SKIM:PbPbEMu+PbPbZEE+PbPbZMM+PbPbZMu,EI,DQM:@commonFakeHLT+@standardDQMFakeHLT --datatier AOD,DQMIO --eventcontent AOD,DQM --data --process reRECO --scenario pp -n 10 --repacked --era Run2_2018_pp_on_AA --customise Configuration/DataProcessing/RecoTLR.customisePostEra_Run2_2018_pp_on_AA --no_exec --filein step1.root --fileout file:step2.root --customise_commands="process.DQMStore.saveByLumi = cms.untracked.bool(True)"

echo "print(process.AlcaBeamMonitor.perLSsaving)" >> step2_test_RAW2DIGI_L1Reco_RECO_ALCA_SKIM_EI_DQM.py

python step2_test_RAW2DIGI_L1Reco_RECO_ALCA_SKIM_EI_DQM.py
```

Reply:
OK, I see your point, thank you!. However, I am not sure how to proceed since this is not an Era related change, any advice? Thanks

Reply:
Modifiers defined in [`Configuration/ProcessModifiers/python`](https://github.com/cms-sw/cmssw/tree/master/Configuration/ProcessModifiers/python) are "technical" like this. So define the Modifier there and use it along `cmsDriver.py --procModifiers=<Modifier>`.